datasets/kwaliteitsmonitor/kwaliteitsmonitorWonenVerblijfsobject/v1.0.0.json

```diff
@@ -0,0 +1,254 @@
+    "display": "id",
+    "properties": {
+      "id": {
+        "type": "integer",
+        "description": "Unieke id van het verblijfobject"
+      },
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "adres": {
+        "type": "string",
+        "description": "Adres van het verblijfobject"
+      },
+      "huisnummer": {
+        "type": "integer",
+        "description": "Huisnummer van het verblijfobject"
+      },
+      "huisletter": {
+        "type": "string",
+        "description": "Huisletter van het verblijfobject"
+      },
+      "huisnummertoevoeging": {
+        "type": "string",
+        "description": "Huisnummertoevoeging van het verblijfobject"
+      },
+      "postcode": {
+        "type": "string",
+        "description": "Postcode van het verblijfobject"
+      },
+      "woonplaats": {
+        "type": "string",
+        "description": "Woonplaats van het verblijfobject"
+      },
+      "bagNagId": {
+        "type": "string",
+        "description": "Unieke nummeraanduiding van het verblijfobject"
+      },
+      "bagNagVolgnummer": {
+        "type": "integer",
+        "description": "Volgnummer van de nummeraanduiding behorende bij het verblijfobject"
+      },
+      "votIdentificatie": {
+        "type": "string",
+        "description": "Unieke aanduiding van het verblijfobject"
+      },
+      "bagGebruiksdoel": {
+        "type": "string",
+        "description": "Gebruiksdoel van het verblijfsobject"
+      },
+      "bagTypeadresseerbaarobject": {
+        "type": "string",
+        "description": "-"
+      },
+      "bagStatus": {
+        "type": "string",
+        "description": "De fase van de levenscyclus van een verblijfobject waarin de betreffenhet verblijfobject zich bevindt"
+      },
+      "bagStatusCode": {
+        "type": "string",
+        "description": "-"
+      },
+      "pndIdentificatie": {
+        "type": "integer",
+        "description": "Unieke identificatie van het pand"
+      },
+      "pndOorspronkelijkbouwjaar": {
+        "type": "integer",
+        "description": "Oorspronkelijke bouwjaar van het pand"
+      },
+      "pndLiggingOmschrijving": {
+        "type": "string",
+        "description": "Ligging van het pand"
+      },
+      "mntMonumentstatus": {
+        "type": "string",
+        "description": "Monumentstatus van het pand"
+	  },		
+      "bsdLigtInBeschermdGebied": {
+        "type": "string",
+        "description": "Ja/Nee indicator welke aangeeft of het verblijfobject in een beschermd stads- of dorpgezicht valt"
+      },
+      "bsdBeschermdStadsdorpsgezicht": {
+        "type": "string",
+        "description": "Tot welk gebied van het beschermd stadsdorpsgezicht het verblijfobject behoort"
+      },
+      "brkVveIsEigendomVve": {
+        "type": "string",
+        "description": "Ja/Nee welke aangeeft of het verblijfsobject eigendom is van de VVE?"
```

Comment:
`boolean`?

Reply:
Het veld is geen boolean in overleg met opdrachtgever.